src/widgets/DataTable/DataTable.js

```diff
@@ -96,6 +96,11 @@ const DataTable = React.memo(({ renderRow, renderHeader, style, data, columns, .
     [numberOfEditableCells]
   );
 
+  const renderItem = useCallback(
```

Comment:
Might be worth indicating when the importance of this memoisation in terms of the render cycle.